packages/contracts/scripts/deploy-scripts/mainnet.sh

```diff
@@ -0,0 +1,53 @@
+### All available deploy options at the time of deployment: ###
```

Comment:
`#!/bin/bash` and also `chmod +x mainnet.sh`

Reply:
Added!